docs/src/InvariantTheory/finite_groups.md

```diff
@@ -101,6 +109,14 @@ with generators
 julia> group(IR)
 Matrix group of degree 3
   over cyclotomic field of order 3
+with 2 generators
+  [0   0   1]
+  [1   0   0]
+  [0   1   0]
+
+  [1   0        0]
+  [0   a        0]
+  [0   0   -a - 1]
```

Comment:
Wouldn't you want an explanation for `a` too in this output? Or is this completely clear from the context?